prboom2/src/m_menu.c

```diff
@@ -3159,6 +3159,7 @@ setup_menu_t gen_video_settings[] = {
   EMPTY_LINE,
   { "Fake Contrast", S_CHOICE, m_conf, G_X, dsda_config_fake_contrast_mode, 0, fake_contrast_list },
   { "OpenGL Light Fade", S_CHOICE, m_conf, G_X, dsda_config_gl_fade_mode, 0, gl_fade_mode_list },
+  { "Extra Lighting", S_THERMO, m_conf, G_X, dsda_config_extra_level_brightness },
```

Comment:
Would make more sense to have both Gamma and Extra Lighting sliders, to not encourage people to use a TAS feature.
So lets not put this option in the menu yet, just keep it as a config.

Ill organize the page better to make space for both sliders sometime in the future